molgenis-data-mapper/src/main/java/org/molgenis/data/mapper/MappingServiceController.java

```diff
@@ -192,15 +217,14 @@ public String removeEntityMapping(@RequestParam String mappingProjectId, String
 	 *            name of the source entity
 	 * @param targetAttribute
 	 *            name of the target attribute
-	 * @param sourceAttribute
-	 *            name of the source attribute
+	 * @param algorithm
+	 *            the mapping algorithm
 	 * @return redirect URL for the attributemapping
 	 */
 	@RequestMapping(value = "/saveattributemapping", method = RequestMethod.POST)
```

Comment:
Guess we should have a convention on camel casing in spring requestMapping value URLs